mage_ai/shared/hash.py

```diff
@@ -68,9 +68,17 @@ def _build(obj, key):
     return reduce(_build, keys, {})
 
 
+def extract_including_blank_values(d, keys):
```

Comment:
Can you just add an optional boolean kwarg `including_blank_values` to the `extract` method if the only difference is None check?